boot/boot.go

```diff
@@ -429,18 +430,18 @@ func SetRecoveryBootSystemAndMode(dev snap.Device, systemLabel, mode string) err
 // UpdateManagedBootConfigs updates managed boot config assets if those are
 // present for the ubuntu-boot bootloader. Returns true when an update was
 // carried out.
-func UpdateManagedBootConfigs(dev snap.Device, gadgetSnapOrDir string) (updated bool, err error) {
+func UpdateManagedBootConfigs(dev snap.Device, gadgetSnapOrDir, cmdlineExtra string) (updated bool, err error) {
```

Comment:
we could split the changes to consume the option from the changes to set the option